src/extension.ts

```diff
@@ -648,6 +648,28 @@ export class Extension implements RunHooks {
       await this._queueWatchRun(new this._vscode.TestRunRequest(testItems), 'items');
   }
 
+  private async _showBrowserForRecording(file: string, project: TestProject, connectWsEndpoint: string) {
+    const fileItem = this._testTree.testItemForFile(file);
+    if (!fileItem)
+      throw new Error(`File item not found for ${file}`);
```

Comment:
What happens when this throws? Does the user see an error? I think it's easy to trigger with non-standard `testMatch`. Let's be careful.

Reply:
good catch. I switched it to returns.